src/composables/locale.js

```diff
@@ -0,0 +1,14 @@
+import { getUnLocale } from '@houlagins/locale'
+
+export default () => scbdCMSLocale() || getUnLocale()
+
+
+function scbdCMSLocale(){
+  if(typeof window === 'undefined') return ''
+
+  const pref = window.document.cookie.replace(/(?:(?:^|.*;\s*)Preferences\s*=\s*([^;]*).*$)|^.*$/, '$1')
```

Comment:
not 100% sure it's right but I think this line much simplier to read
```javascript
const locale = (/\bPreferences\s*=\s*Locale\s*=\s*(\w+)\b/.exec(window.document.cookie||'') || [])[1];
```

Reply:
Both not easy for me.  Will use regexer :-)